elife/composer.sls

```diff
@@ -75,7 +82,7 @@ update-composer:
         - name: composer self-update
         - onlyif:
             - which composer
-        - require:
+        - require_in:
```

Comment:
Yeah, it installed then immediately checked if it's up to date. (Don't think there's another way to only run the state if other state was skipped.)